### PR TITLE
✨ Add the Connection Settings screen

### PR DESCRIPTION
**Pull request by zmckinnon**
image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ async function loadResourcesAsync() {
   await Promise.all([
     Asset.loadAsync([]),
     Font.loadAsync({
-      'Roboto': require('./assets/fonts/Roboto-Regular.ttf'),
+      'RobotoRegular': require('./assets/fonts/Roboto-Regular.ttf'),
```
Comment (zmckinnon) [inline src/App.jsx]: We need to clean this up.  We essentially need different Font names for each weight OR we need to figure out how to load all weights under one .tff file.  When I downloaded the fonts from Google Fonts, it had them all as separate files.

File: src/screens/ContactDetailsScreen.jsx [hunk 2]
```diff
@@ -102,29 +182,65 @@ class ContactDetailsScreen extends React.Component {
...
+                    </Button>
+                    <SettingsButton onPress={this.showSettingsModal}>
+                        <SvgWrapper>
+                            <Gear />
+                        </SvgWrapper>
+                        <ButtonText>Settings</ButtonText>
+                    </SettingsButton>
+                </ActionsSection>
+            </Container>
+            <CustomModal
```
Comment (zmckinnon) [inline src/screens/ContactDetailsScreen.jsx]: One day we need to refactor this modal to be reusable.  I may do that real soon.

**Comment by irtriskit** [comment]
This isn't currently saving this data, right? With connections being saved on the user object, we may want to think about splitting that out maybe?

**Comment by zmckinnon** [comment]
> This isn't currently saving this data, right? With connections being saved on the user object, we may want to think about splitting that out maybe?

Yeah, this is supposed to just be styling.  I have definitely been thinking about splitting out connections.  Let's talk about it the next time we meet.

**Pull request merged**